### ISSUE TITLE
Description of rule W0221 not complete

### ISSUE THREAD

**Issue by pauljansen42** [PyCQA/pylint#5875]
The description of rule W0221 is not complete. Currently it is: "Used when a method has a different number of arguments than in the implemented interface or in an overridden method". However, the rule is also fired in case the argument names differ. So it is better to change the description into:

"Used when a method has a different number of arguments or different argument names than in the implemented interface or in an overridden method."

or something in that style.

**Comment by jacobtylerwalls** [comment]
Hi @pauljansen42 thanks for this. Do you have an example to share? Right now I'm not seeing it.

```python
class A:
    def z(self, first): ...

class B(A):
    def z(self, second): ...
```
emits W0237 instead:
```
a.py:5:4: W0237: Parameter 'first' has been renamed to 'second' in overridden 'B.z' method (arguments-renamed)
```

**Comment by pauljansen42** [comment]
Hi @jacobtylerwalls, thanks for your answer. I am actually using pylint 1.9.5. I can't upgrade to a newer pylint version because my company is still using Python 2. With your example I get the confusion violation message:

  W0221:  5,4: B.z: Parameters differ from overridden 'z' method

So probably this is because I am using an old pylint version. If you can confirm this, then you can close this ticket. Thanks for your time.

**Comment by jacobtylerwalls** [comment]
Yes, I was testing on the main branch for Python 3.6.2+. Thanks for following up.
